addons/binding/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostal/inverter/WebscrapeHandler.java

```diff
@@ -98,11 +103,11 @@ private Document getDoc() throws IOException {
         return Jsoup.connect(config.url).header("Authorization", "Basic " + base64login).get();
     }
 
-    private State getState(String value) {
-        try {
-            return new DecimalType(new BigDecimal(value));
-        } catch (NumberFormatException e) {
```

Comment:
Will this exception not occur, or was it only because of `null`

Reply:
I am not sure. I assume value can be null too. Or contain a String which could not be parsed. But to catch the NFE and return a StringType did not make sense as well. The receiving item may be a Number.

A proper solution would be to catch the   NFE and return an UndefType.UNDEF instead.